docs/source/markdown/podman-create.1.md

```diff
@@ -706,6 +706,12 @@ Tune the host's OOM preferences for containers (accepts -1000 to 1000)
 #### **--os**=*OS*
 Override the OS, defaults to hosts, of the image to be pulled. For example, `windows`.
 
+#### **--personality**=*persona*
+
+Personality sets the execution domain via linux personality.
+Reference: https://man7.org/linux/man-pages/man2/personality.2.html
+Example: **--personality=LINUX32**.
```

Comment:
Although small, to be consistent, this should be in the Examples section at the bottom of the man page.